src/lib/tw-join.ts

```diff
@@ -8,7 +8,7 @@
  * Original code has MIT license: Copyright (c) Luke Edwards <luke.edwards05@gmail.com> (lukeed.com)
  */
 
-export type ClassNameValue = ClassNameArray | string | null | undefined | 0 | false
+export type ClassNameValue = ClassNameArray | string | null | undefined | 0 | false | bigint
```

Comment:
Only falsy bigints should be allowed here. Could you change it to only allow `0n` instead?